src/main/java/core/basesyntax/LinkedListIndexOutOfBoundsException.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public class LinkedListIndexOutOfBoundsException extends Throwable {
```

Comment:
you should extend the
1 Exception if you want checked exception
2 RuntimeException if you want unchecked exception

Throwable is bad idea to inherit.